login/login.default.html

```diff
@@ -49,7 +49,7 @@
                     <div class="mb-3">
                         <z-login-submit-button button-text="Log in" text-color="#FEF6E4" class="d-grid gap-2"></z-login-submit-button>
                     </div>
-                    <p class="text-center">Powered by <a href="https://www.zuar.com/">Zuar</a></p>
+                    <p class="text-center" style="font-size:12px;font-style: italic;">Powered by <a href="https://www.zuar.com/">Zuar</a></p>
```

Comment:
Can we use bootstrap classes here instead of inlined css?

Reply:
@dylanspurgin 
Smallest font size class that provide Bootstrap is fs-6 which is equivalent of 16px (1rem), so, unfortunately, I found inline styling as the only option
(https://getbootstrap.com/docs/5.0/utilities/text/#font-size)